src_updater/updater.py

```diff
@@ -98,7 +98,7 @@ def check_if_update_possible(self):
             #if the update option isn't present, the default is to do updates.
             if update_option_name not in self.options or self.options[update_option_name]:
                 # check if github has a newer version than us
-                latest = "https://api.github.com/repos/Hyphen-ated/RebirthItemTracker/releases/latest"
+                latest = "https://api.github.com/repos/rchardon/RebirthItemTracker/releases/latest"
```

Comment:
In a pull request, you shouldn't change references pointing to the main repository. This cannot be merged while this is present.

Also, sneakingly changing the update source is a pretty malicious action.

Reply:
How is it sneaky ? Hyphen told me to do that

Reply:
You would normally do that in a separate branch if you want to maintain your own releases, not in the pull request branch.

Had not this been spotted, the next version would have switched the updater to use your releases, which is a very bad thing for unsuspecting users.

Reply:
Hey it's okay, nobody was being malicious here. I decided I was not going to merge any of this, I told rchardon that they should probably start maintaining a new fork, and I mentioned that this line would have to be changed. When that change was committed, it was reflected in this "PR branch" because it's just a PR from the branch where development was happening. I'm not going to have autoupdates go from my fork to the new fork: users will have to take an explicit step of trusting the new maintainer by going and downloading the new program.